src/database/models/index.js

```diff
@@ -34,3 +34,5 @@ Object.keys(db).forEach((modelName) => {
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
```

Comment:
is this line necessary

Reply:
yes very necessary because resetToken is needed and I saw in some codes you remove it please do not remove it again because once I push and you pull and delete it my resetPassword endpoint won't work